### PR TITLE
chore(1821) - Add feedback button and another minor changes

### LINKED ISSUE #1821: [Chore]: Add feedback button and another minor changes
## Is your chore related to a problem? Please describe:

- No way to provide feedback

## Describe the solution you'd like:

- Add feedback component

## Describe alternatives you've considered:

- None

## Additional context:

- N/A

### PR DESCRIPTION
**Pull request by hybridx**
## Closes / Fixes / Resolves

- Resolves #1821 

## Explain the feature/fix

- Added banner as info

## Does this PR introduce a breaking change

- No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

Screenshot 2022-08-08 at 7 14 20 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/pages/index.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import Body from "../components/layout/body";
 import { ComponentWithAuth } from "../utils/auth.utils";
+// import '@one-platform/opc-feedback/dist/opc-feedback';
```
Comment (soumyadip007) [inline packages/manager/pages/index.tsx:3]: can you please remove this, if we are not using this in future. If not then it's okay

File: packages/manager/pages/index.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
 const Home: ComponentWithAuth = () => {
   return (
     <Body title={"Welcome to SPAship! 👋 "} breadcrumbs={[{ path: "/", title: "Home" }]}>
       <div className="page-body">To get started select an option from the left sidebar!</div>
+      {/* <opc-feedback></opc-feedback> */}
```
Comment (soumyadip007) [inline packages/manager/pages/index.tsx:9]: Same for this

**Review by soumyadip007 — APPROVED** [review]

**Review by arkaprovob — APPROVED** [review]
If possible, please remove the packages/manager/package-lock.json file from the commit.

**Review by soumyadip007 — APPROVED** [review]

File: packages/manager/pages/index.tsx (hunk 2 above)
Comment (hybridx) [reply packages/manager/pages/index.tsx:9]: We are going to use this

**Pull request merged**